react/features/recording/components/web/RecordingLabel.tsx

```diff
@@ -52,14 +52,19 @@ class RecordingLabel extends AbstractRecordingLabel {
         }
 
         // @ts-ignore
-        const { classes, mode } = this.props;
+        const { classes, mode, t } = this.props;
+        const isRecording = mode === JitsiRecordingConstants.mode.FILE;
+        const icon = isRecording ? IconRecord : IconSites;
+        const tooltipKey = isRecording ? t('videoStatus.recording') : t('videoStatus.streaming');
```

Comment:
Either leave the key or change the variable name to `tooltip` and remove the `t` call below